core/src/main/groovy/noe/tomcat/configure/SecureHttpConnectorTomcat.groovy

```diff
@@ -25,19 +31,55 @@ public class SecureHttpConnectorTomcat extends ConnectorTomcatAbstract<SecureHtt
   private String sslProtocol
   private String keystoreFile
   private String keystorePass
+  private String keystoreType
   private boolean clientAuth
+  private String truststoreFile
+  private String truststorePass
+  private String truststoreType
+  private String sslImplementationName
+  private boolean sslEnabledProtocols
 
   // SSL APR
+  private String sslCACertificateFile
   private String sslCertificateFile
   private String sslCertificateKeyFile
   private String sslPassword
+
+  // Inner elements
+  private ConnectorUpgradeProtocolTomcat upgradeProtocol
   // - ^^^ ------------------------------------------------------
 
   public SecureHttpConnectorTomcat() {
     super.setSecure(true)
     sslEnabled = true
   }
 
+  /**
+   * Configure secure http connector to expect certificates in ${SYSTEM_TEMP}/ssl/self_signed directory
+   * Expected names:
+   *  <ul>
+   *    <li>certificate = server.crt</li>
+   *    <li>key file = server.key</li>
+   *    <li>keystore = server.jks</li>
+   *    <li></li>
+   *  </ul>
+   *  Password for keystore, trustore and SSL sets to "changeit" (without apostrophes).
+   */
+  SecureHttpConnectorTomcat setDefaultCertificatesConfiguration() {
+    String sslStringDir = new File(new Platform().getTmpDir(), new PathHelper().join("ssl", "self_signed")).getCanonicalPath()
```

Comment:
There is going too much on 1 line, I would change
````
String sslStringDir = new File(new Platform().getTmpDir(), new PathHelper().join("ssl", "self_signed")).getCanonicalPath()
````
Into
````
File sslRoot = new File(new Platform().getTmpDir(), "ssl" )
String sslStringDir = new File(sslRoot, "self_signed").getCanonicalPath()
`````

````

Reply:
I agree.Just make the change and wait for your opinion :)